github/ci/prow-deploy/files/jobs/kubevirt/common-templates/common-templates-periodics.yaml

```diff
@@ -1,6 +1,6 @@
 periodics:
 - name: periodic-common-templates-nightly-fedora-cdi-builder
-  cron: "30 2 * * *"
+  cron: "30 14 * * *"
```

Comment:
Given that these tests require a bare metal it is better to execute them when there is less load in the system.

Reply:
Done